### PR TITLE
adding Codefresh to vocabulary.md

### PR DESCRIPTION
**Pull request by anais-codefresh**
This commit provides the information on the vocabulary used within Codefresh

### CODE REVIEW

File: docs/vocabulary.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 - [Introduction](#introduction)
 - [Terminology Used by CI/CD Tools and Technologies](#terminology-used-by-cicd-tools-and-technologies)
   - [ArgoCD](#argo-cd)
+  - [Codefresh](#codefresh)
```
Comment (fdegir) [inline docs/vocabulary.md:6]: Sorry for nitpicking but I'm wondering if you could move Codefresh after CircleCI to continue listing technologies/projects alphabetically?

File: docs/vocabulary.md [hunk 2]
```diff
@@ -75,6 +76,17 @@ Some of the core ArgoCD concept are listed below: [[16]]
 - **Sync phases** [[18]]: Pre-sync, sync and post-sync allow for controlled orchestrated execution of sync operations.
 - **Sync waves**: Within any one sync phase, an additional control of sequential execution is accomplished using waves, to ensure certain resources are healthy before subsequent resources are synced.
 
+### Codefresh
```
Comment (fdegir) [inline docs/vocabulary.md:79]: Same here. Please move Codefresh after CircleCI.

File: docs/vocabulary.md [hunk 3]
```diff
@@ -362,6 +374,7 @@ pipeline in corresponding documentation.
 | Project            |           |            |          |          |         |                        |
 |--------------------|-----------|------------|----------|----------|---------|------------------------|
 | **ArgoCD**         | Sync Wave | Sync Phase | Sync     | N/A      | Event   | Application Controller |
+| **Codefresh**      | N/A       | Step       | Stage    | Pipeline | Trigger | Runtime environment    |
```
Comment (fdegir) [inline docs/vocabulary.md:377]: and here too.

**Review by fdegir — CHANGES_REQUESTED** [review]
Thanks for including Codefresh in the document. Have minor comments inline.

File: docs/vocabulary.md (hunk 1 above)
Comment (anais-codefresh) [reply docs/vocabulary.md]: Not nitpicking, moved it further down

File: docs/vocabulary.md (hunk 3 above)
Comment (anais-codefresh) [reply docs/vocabulary.md]: Done

File: docs/vocabulary.md (hunk 2 above)
Comment (anais-codefresh) [reply docs/vocabulary.md]: Yes, sorry about that

**Review by fdegir — APPROVED** [review]
Thank you!

**Pull request merged**